plugins/base_commands.py

```diff
@@ -8,8 +8,12 @@
 
 import datetime
 
+import logging
 from spockbot.plugins.base import PluginBase
 from spockbot.vector import Vector3
+import json
+tpareqs = {}
```

Comment:
if tpareqs is just used in the module, it should be a class level object. as long as you just modify it, all instances from the class will have access to it. If overwritten, just the object will loose the reference to it. Be carefull with the lifetime of the object, it will never be destroyed!